### ISSUE TITLE
Donjon BC / le Méchanar NM et HM / Couloir vide de mobs

### ISSUE THREAD

**Issue by Notil78** [way-of-elendil/3.3.5#917]
Bonjour,

Le couloir entre Néantomancienne Sepethrea (https://woehead.way-of-elendil.fr/?npc=19221) et le boss de fin Pathaleon le Calculateur (https://woehead.way-of-elendil.fr/?npc=19220) est complétement vide de NPC.
Par exemple : https://www.youtube.com/watch?v=Wk3IwnlpH_Y   a 52"20.

Merci,
Bonne journée.

**Comment by Killyana** [comment]
C'est pas des spawn mais des mobs invoqués https://youtu.be/mgAvgObSjKM?t=216

**Comment by joshwhedon** [comment]
@Killyana  y'a aucun areatrigger, aucun trigger, rien du tout dans ce couloir. Aucune idee comment ces invocations sont declenchees. T'as des infos ?

**Comment by Killyana** [comment]
Non, je suis arrivé à la même conclusion, il semble que la mort du pack qui garde ce couloir soi le déclencheur de l'évent, même le boss est invoqué

**Comment by joshwhedon** [comment]
Hum, possible. Du coup je peux probablement le scripter comme ca.

**Comment by Ferro3** [comment]
Mes souvenirs sont pas super, mais j'ai la réponse, c'est en avançant que les mobs sont déclenchés.
- le 1er est au début du couloir
- le second à 1/3 environ
- et le dernier au 2/3 environ

Par contre le nombre de mob et le type je ne me rappelle plus du tout les détails, mais il y a des, elfs de sang et un élémentaire à chaque pop.
Il me semble aussi qu'il y a 2 ou 3 vagues par déclencheur, mais là, je suis plus du tout sûr.

Par contre, je peux vous dire que c'est un passage délicat à passer au lv70, surtout en HM ; d'où mes souvenirs :D
Le mieux serait de vérifier avec un compte officiel, si l'instance n'a pas bougée.
Bon courage.
